pep-0701.rst

```diff
@@ -403,6 +403,13 @@ Here are some of the arguments in favour:
     def py2c(source):
         return f"{source.removesuffix(".py")}.c"
 
+* Code generators (like `ast.unparse <https://docs.python.org/3/library/ast.html#ast.unparse>`_ from standard library) in their
```

Comment:
FYI, instead of a hardcoded URL, you can instead use an actual Sphinx reference via Intersphinx, which will render with the appropriate formatting and won't break if any part of the URL, page or anchor name changes, plus is a lot more concise, i.e. just `` :function:`ast.unparse` ``